src/ControlSystem/ControlErrors/Translation.cpp

```diff
@@ -0,0 +1,11 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "ControlSystem/ControlErrors/Translation.hpp"
+
+#include <cstddef>
+
+namespace control_system::ControlErrors {
+template struct Translation<1_st>;
+template struct Translation<2_st>;
```

Comment:
These don't do anything useful because the class is entirely defined in the hpp.